src/components/ThumbnailCanvasGrouping.js

```diff
@@ -72,10 +72,12 @@ export class ThumbnailCanvasGrouping extends PureComponent {
           )}
         >
           {currentGroupings.map((canvas, i) => (
-            <CaptionedCanvasThumbnail
+            <IIIFThumbnail
               key={canvas.id}
-              canvas={canvas}
-              height={(position === 'far-right') ? style.height - (1.5 * SPACING) : height - (1.5 * SPACING)}
+              resource={canvas}
+              labelled
+              maxHeight={(position === 'far-right') ? style.height - (1.5 * SPACING) : height - (1.5 * SPACING)}
```

Comment:
I'm wondering if the change from height -> maxHeight has anything to do with this?

![Kapture 2020-05-19 at 15 25 53](https://user-images.githubusercontent.com/1656824/82380277-69371700-99e5-11ea-8883-6deb6da9fdab.gif)

Reply:
Fixed; labels weren't being clamped in quite the right place for the right nav.